src/main/edu/colostate/cs/cs414/method_men/jungle/client/gui/GUI.java

```diff
@@ -51,15 +58,15 @@ public void changePageTo(JPanel comp) {
         repaint();
         pack();
     }
-    public void startGUI(GUI g, Socket client){
+    public void startGUI(GUI g, Socket socket){
```

Comment:
```suggestion
    public void startGUI(GUI g){
```
Change it here as well

Reply:
I made that change and I also moved the login authentication code to the server. take a look when you get a chance